abletonosc/clip.py

```diff
@@ -70,27 +70,49 @@ def clip_callback(params: Tuple[Any]) -> Tuple:
             "remove_notes_by_id"
         ]
         properties_r = [
+            ##"end_time" ## doesn't seem to do anything, can't test
             "file_path",
             "gain_display_string",
+            ##"groove", ## if other than None, says "Error handling OSC message: Infered arg_value type is not supported"
+            "has_groove", ## False/True
+            ## is_arrangement_clip - wasn't able to get this to do anything
             "is_midi_clip",
             "is_audio_clip",
+            "is_overdubbing", ## False/True
             "is_playing",
             "is_recording",
+            ##"is_triggered", ## if clip was triggered to record, or is recording
+            ## the above does not seem to actually do anything
             "length",
-            "playing_position"
+            "playing_position",
+            "sample_length", ## New, works - AudioClips only, -1 if no sample
+            "start_time", ## New, works
+            ##"view", ##"Infered arg_value type is not supported"
+            ##"warp_markers", ## "Infered arg_value type is not supported"
+            "will_record_on_start" ##False/True
         ]
         properties_rw = [
             "color",
+            "color_index", ## is the range 0 to 69?
```

Comment:
Yes it is, just tested. This comment can be removed as you've also documented the range in the README